aliases/available/git.aliases.bash

```diff
@@ -196,3 +194,17 @@ esac
 function gdv() {
 	git diff --ignore-all-space "$@" | vim -R -
 }
+
+function gcom() {
+	if git branch | grep -q main; then
```

Comment:
More context on the `-q` flag:
https://stackoverflow.com/questions/56170215/what-is-the-point-of-grep-q

Reply:
grep's man page:
https://www.man7.org/linux/man-pages/man1/grep.1.html